Modules/Scripted/Endoscopy/Endoscopy.py

```diff
@@ -185,11 +187,12 @@ def setupFlythroughUI(self):
 
         # View angle slider
         viewAngleSlider = ctk.ctkSliderWidget()
+        # View angles are in degrees
```

Comment:
After renaming the variable, remove this comment.

Reply:
Will do.

Reply:
```suggestion
```